documentation-template (#342)
vtex/faststore: pull_request_template.md

## What's the purpose of this pull request?
<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?

> <em>Describe the steps with bullet points. Is there any external reference, link, or example?</em> 

Reviewer: What would be the difference between the reference requested here and the one in "References" section? 
It might be better to use another nomenclature or explain it so the dev don't get confused as if we're asking for the same thing in 2 different sections

> <em>Describe the steps with bullet points. Is there any extenal link that can be used to better test it or an example?</em> 

Reviewer: ```suggestion
<em>Describe the steps with bullet points. Is there any external link that can be used to better test it or an example?</em> 
```

## References

> <em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>  

Reviewer: ```suggestion
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  
```

> <em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em> 

Reviewer: ```suggestion
<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em> 
```
